pkg/arvo/app/graph-store.hoon

```diff
@@ -513,7 +513,7 @@
     =/  =ship  (slav %p i.t.t.path)
     =/  =term  i.t.t.t.path
     =/  =index:store
-      (turn t.t.t.t.path |=(=cord (slav %ud cord)))
+      (turn t.t.t.t.path |=(=cord (rash cord dem:ag)))
```

Comment:
Why?

Reply:
To avoid the dots in the `@ud` syntax i.e. `1.000.000`

Reply:
Wait, but this is in the scry path? I don't think we should start introducing non-native syntax/encodings into those. If the concern here is generating scry URLs on the client side, implementing `scot` in "airlock" libraries is perfectly reasonable.